fixtures/exec-git-checkout-test.yaml

```
@@ -0,0 +1,47 @@
+apiVersion: configs.flanksource.com/v1
+kind: ScrapeConfig
+metadata:
+  name: exec-git-checkout-test
+  namespace: default
+spec:
+  schedule: "@every 15m"
+  exec:
+    - name: run-inline-script-from-repo
+      type: $.type
+      id: $.id
+      name: $.name
```

YAMLlint (1.37.1)

[error] duplication of key "name" in mapping

(key-duplicates)